mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -33,11 +33,10 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.{DataFrame, Dataset, Row}
 import org.apache.spark.sql.functions.{col, udf}
 import org.apache.spark.sql.types.{IntegerType, StructType}
-import org.apache.spark.util.VersionUtils.majorVersion
-
+import org.apache.spark.storage.StorageLevel
 /**
  * Common params for KMeans and KMeansModel
- */
+ */ 
```

Comment:
I cleaned the code, then I run /dev/lint-scala and I received : 
Scalastyle checks passed.